Bugsnag/BugsnagSessionTracker.h

```diff
@@ -49,9 +43,7 @@ static NSNotificationName const BSGSessionUpdateNotification = @"BugsnagSessionC
 - (void)startNewSessionIfAutoCaptureEnabled;
 
 /**
- Update the details of the current session to account for externally reported
- session information. Current session details are included in subsequent crash
- reports.
+ Used by bugsnag-unity
```

Comment:
This is useful, though the old comment explained what this method actually does. "externally reported" always meant "libraries built on top" so the comment could be expanded a bit to be more specific.

Reply:
Hmm I did start rewriting this and then saw that I was essentially repeating the name of the method 🤔

Reply:
Yeah I could see that. Though it might be otherwise lost to time _why_ this would ever be done. What we probably need is a unified place for these kinds of hooks for separate libraries. I think @lemnik had some ideas on this since Android has a similar issue.

Reply:
Yeah I was thinking of some kind of attribute we could use to tag APIs used be other notifiers.

Refactoring is so much harder when other code depends on internals 😬